taf/git.py

```diff
@@ -101,9 +104,14 @@ def __init__(
         self.default_branch = default_branch
 
     _pygit = None
-
+    
```

Comment:
Please run the black formatter, or the CI will fail. See https://www.geeksforgeeks.org/python-code-formatting-using-black/